enterprise-suite/templates/es-grafana.yaml

```diff
@@ -84,7 +84,7 @@ spec:
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: grafana-server
+  name: grafana
```

Comment:
Does line 106 need to change too?  
    `name: grafana-server`
What about the comment at line 11?
    `# 2. add prometheus data source to grafana-server.`